### ISSUE TITLE
support function argument validation

### ISSUE THREAD

**Issue by florianschanda** [florianschanda/miss_hit#60]
https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html
```
function out = myFunction(A, B, C)   
    arguments
        A (1,1) string 
        B (1,:) double
        C (2,2) cell
    end

    % Function code
    ...
end
```

Seems similar what you can do inside class `properties`

Open question:
* Is this valid only inside classdef or can you do that outside?

**Comment by wbekerom** [comment]
Hi @florianschanda,

I noticed that **mh_style** in miss_hit version 0.9.39 does not allow an empty "validation function" list. MATLAB R2021b does not explain about this syntax, so one would assume it is valid.

For this snippet:
```
function out = my_function(A, B, C)
    arguments
        A (1, 1) string {}
        B (1, :) double {}
        C (1, 4) cell {}
    end

end
```

**mh_style** reports:
```
In test_mh_style/my_function.m, line 3
|         A (1, 1) string {}
|                          ^ error: expected IDENTIFIER, found C_KET instead
```

**Comment by florianschanda** [comment]
OK thanks, I will have a look

**Comment by florianschanda** [comment]
Looks like my parser expects a non-empty list; this is easy to fix.

**Comment by florianschanda** [comment]
@wbekerom this is fixed in 0.9.41 which I have just released. Thanks for the report ;)

**Comment by wbekerom** [comment]
Thanks a lot for the ⚡ fast fix!
I can confirm that the bug is indeed fixed in 0.9.41
